ocs_ci/ocs/ceph_debug.py

```diff
@@ -1,8 +1,8 @@
 import logging
 
 from ocs_ci.framework import config
+from ocs_ci.helpers.odf_cli import ODFCliRunner
```

Comment:
Dont forget to [download](https://github.com/red-hat-storage/ocs-ci/blob/f07e81498aeca107532ab54e40752140826883f4/ocs_ci/helpers/odf_cli.py#L17) the binary before using it